src/assets/source/styles/partials/_serlo_home.scss

```diff
@@ -16,8 +16,8 @@ $homeBreakpoint-lg-min: $homeBreakpoint-lg;
 $homeBreakpoint-lg-max: ($homeBreakpoint-lg-min - 1);
 
 $homeBreakpoint-huge: 1200px;
-$homeBreakpoint-huge-min: $homeBreakpoint-lg;
-$homeBreakpoint-huge-max: ($homeBreakpoint-lg-min - 1);
+$homeBreakpoint-huge-min: $homeBreakpoint-huge;
```

Comment:
Any reason that you rename that to `-huge` instead of `-lg`? Initially, the naming was analogously to Bootstrap.